microsetta_private_api/repo/admin_repo.py

```diff
@@ -1015,7 +1043,71 @@ def _generate_novel_barcodes(self, number_of_kits, number_of_samples,
                     kit_name_and_barcode_tuples_list.append(
                         (name, new_barcodes[offset + i]))
 
-        return kit_name_and_barcode_tuples_list, new_barcodes
+            return kit_name_and_barcode_tuples_list, new_barcodes
```

Comment:
Was the change of scope consequential? I wouldn't expect it to be? 

```suggestion
        return kit_name_and_barcode_tuples_list, new_barcodes
```